WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2419,6 +2423,10 @@ public boolean updatePostContentNewEditor(boolean isAutoSave, String title, Stri
       * URLs will have been replaced with the remote ones.
      */
     private boolean compareCurrentMediaMarkedUploadingToOriginal(String newContent) {
```

Comment:
I know it's not part of the diff but, since we're touching this method, how about we rename it to something what would suggest a yes/no question? Currently, the name is `compare....` but it returns true or false, which doesn't explain what true means or what false means. How about something like `shouldMplaMpla....` the answer of which will indeed be yes/no?

Reply:
totally! made the change in aa60c777f6